setup.py

```diff
@@ -10,7 +10,7 @@
 
 name = 'csv-wrangler'
 package = 'csv_wrangler'
-description = 'Statically-typed CSV helpers'
+description = 'Statically typed Python 3 CSV generation helpers'
```

Comment:
I _think_ this should still by hyphenated?

Reply:
sure